### PR TITLE
feat: [CO-784] add documentation URL attributes

### PR DESCRIPTION
**Pull request by inno-dima**
**What has changed:**

- Added LDAP attributes in attrs.xml and rights-domainadmin.xml
- Added JSON configuration for LDAP update script

### CODE REVIEW

**Review by frisonisland — APPROVED** [review]

File: store/ldap/src/updates/attrs/1690904855.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "zimbra_globalconfig": [
```
Comment (keshavbhatt) [inline store/ldap/src/updates/attrs/1690904855.json:2]: We don't need migration, since these are new attributes.

**Review by keshavbhatt — CHANGES_REQUESTED** [review]
Reuqested a change everything else look good.

File: store/ldap/src/updates/attrs/1690904855.json (hunk 1 above)
Comment (frisonisland) [reply store/ldap/src/updates/attrs/1690904855.json:2]: Are you sure? The documentation suggest the opposite. I thought the update was meant to add a default value to new attributes created on existing entities
See: https://github.com/zextras/carbonio-mailbox/blob/devel/store/ldap/src/updates/attrs/ReadMe.md

**Review by Polpetta — APPROVED** [review]
Everything is fine after the migration thing with LDAP is settled

File: store/ldap/src/updates/attrs/1690904855.json (hunk 1 above)
Comment (keshavbhatt) [reply store/ldap/src/updates/attrs/1690904855.json:2]: Yes its true. But as discussed these attributes will probably not have any default value.

Comment (frisonisland) [reply store/ldap/src/updates/attrs/1690904855.json:2]: It was confirmed we have a default value, but we have to wait for it to be provided!

**Review by keshavbhatt — CHANGES_REQUESTED** [review]
You need to build the project after updating attrs.xml and commit whatever changed, usually the build will generate and update some files. They need to be commited.

Other than this, I asked for minor cleanup of ReadMe file which was updated.

**Review by keshavbhatt — APPROVED** [review]

**Review by aheeva-yuliya — APPROVED** [review]

**Pull request merged**